rook/storageclass.yaml

```diff
@@ -0,0 +1,33 @@
+# See https://rook.io/docs/rook/v1.5/ceph-pool-crd.html
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: replicapool
+  namespace: rook-ceph
+spec:
+  failureDomain: host
+  replicated:
+    size: 2
```

Comment:
This must also match be maximum the nr of worker nodes right? Maybe add it to the README

Reply:
No, I think this is just how many times ceph should replicate the data on volumes with this storage class.
Maybe I'm wrong, but I've been running like this anyway so it's not a "must" at least :slightly_smiling_face:

Reply:
Yea, I might be completely wrong here. I just remembered fiddling around with some of these numbers to get one cluster setup to work. But it must've been the mons then